bpmn-model/src/test/java/io/camunda/zeebe/model/bpmn/validation/ExpectedValidationResult.java

```diff
@@ -64,7 +64,7 @@ public boolean matches(final ValidationResult result) {
       match = false;
     }
 
-    if (!expectedMessage.equals(result.getMessage())) {
+    if (!result.getMessage().startsWith(expectedMessage)) {
```

Comment:
Why was this necessary?

Reply:
It was not necessary. But I aligned the class with the one from the engine module. 
We have this class in both models :sweat_smile:

Reply:
ok thanks for explaining